articles/quickstart/webapp/django/01-login.md

```diff
@@ -174,13 +178,18 @@ LOGIN_REDIRECT_URL = '/dashboard'
 
 ## Trigger Authentication
 
-Add a handler for the `index` view in your `views.py` to render the `index.html`
+Add a handler for the `index` view in your `views.py` to render the `index.html` if the user needs to log in. If the user is already logged in, the `dashboard` view will be shown instead.
```

Comment:
> the `dashboard` view will be shown instead

Does `dashboard` here refer to a code artifact, like a property name or variable? If not, it's better to put this in normal double quotes rather than backticks to reduce confusion (or **bold** if you can consider a view as  a UI element).

[Style Guide - formatting](https://github.com/auth0/docs/blob/master/STYLEGUIDE.md#formatting)

Reply:
It's a "view" or "page" the user can navigate to. Just like "index", which is also mentioned in the paragraph. I kept the same format that was used for "index". Do you think I should change this as well?